src/domain/collaboration/callout/callout.service.ts

```diff
@@ -466,7 +468,7 @@ export class CalloutService {
   ): Promise<ICalloutContribution> {
     const calloutID = contributionData.calloutID;
     const callout = await this.getCalloutOrFail(calloutID, {
-      relations: { contributionPolicy: true },
+      relations: { contributions: true },
```

Comment:
Are you sure we don't need contributionPolicy anymore?

Reply:
ah, good catch it seems this was there for the creation and it should stay, I'll double-check and push it back